gradio/interface.py

```diff
@@ -486,7 +485,8 @@ def render_flag_btns(flagging_options):
 
                     with Column(variant="panel"):
                         for component in self.output_components:
-                            component.render()
+                            if not (isinstance(component, State)):
```

Comment:
Out of curiosity - this was a pre-existing bug?

Reply:
Originally we would unnecessarily create a second `State` component with the identical properties as the first. It didn't break anything, but better to avoid it imo.